pkg/server/chaosd/network.go

```diff
@@ -79,11 +79,9 @@ func (networkAttack) Attack(options core.AttackConfig, env Environment) (err err
 				return perrors.WithStack(err)
```

Comment:
We can delete `NeedApplyIptables()` and `NeedApplyIPSet` directly.

Reply:
I think leaving them can make the code logic clearly, and they may be needed to add features later (by the way, `NeedApplyIPSet()` is working now)

Reply:
I think the logic will be clearer If we delete them. The logic: Chaos Mesh will reset all network action when you apply the network actions. This logic will be more simple.

Reply:
OK, I think it's OK to be the same as chaos mesh, `NeedApplyIptables` can be deleted, but I find `chaosd attack network delay -l 1s -d eth0` will not work without `NeedApplyIPSet()`